### ISSUE TITLE
Bug(Sprint Poker): Error in ParabolScopingSearchResultItem

### ISSUE THREAD

**Issue by Dschoordsch** [ParabolInc/parabol#7943]
https://parabol.sentry.io/issues/3402262694/?referrer=github_plugin

```
TypeError: Cannot destructure property 'id' of 'u' as it is undefined.
  at Ft (./packages/client/components/ParabolScopingSearchResultItem.tsx:78:14)
  at Set.forEach (<anonymous>)
...
(31 additional frame(s) were not displayed)
```

## How to reproduce
* Create a Sprint Poker meeting
* add parabol tasks
* advance the meeting
* delete one of the tasks (the task itself from the task dashboard by removing its content)
* go back to the sprint meeting scoping session
* press the new task button

**Comment by Dschoordsch** [comment]
2.2k events, 355 users, thus this should be a p1

**Comment by Dschoordsch** [comment]
Sentry issue: [ACTION-PRODUCTION-E0T](https://parabol.sentry.io/issues/3987304563/?referrer=github_plugin)

**Comment by Dschoordsch** [comment]
Sentry issue: [ACTION-PRODUCTION-DWZ](https://parabol.sentry.io/issues/3983042913/?referrer=github_plugin)

**Comment by Dschoordsch** [comment]
Sentry issue: [ACTION-PRODUCTION-DWY](https://parabol.sentry.io/issues/3983042911/?referrer=github_plugin)

**Comment by Dschoordsch** [comment]
Linked additional issues which have different backtrace but appear in the same area and timespan thus probably have the same root cause.

**Comment by Dschoordsch** [comment]
Yes. I discussed this yesterday with Matt. It's an issue resulting from adding the optimistic task to the edges of the task search result. Omitting the optimistic task would work, but Matt also suggested rearranging the GraphQL fragments, because James recently found out that something similar happened because of @plural items nested in a fragment.

**Comment by lilicaruso** [comment]
Enterprise [User](https://app.hubspot.com/live-messages/3888472/inbox/4222831376#email) 🔒 followed up on resolution of this one.

**Comment by Dschoordsch** [comment]
The workaround should be released by the end of this week.

**Comment by Dschoordsch** [comment]
Just removed the p1 label to not convey unnecessary urgency since the workaround solves it for now.

**Comment by jmtaber129** [comment]
@Dschoordsch @BartoszJarocki can this be closed in favor of a task for the actual fix since the bandaid fix is merged?

**Comment by Dschoordsch** [comment]
Sentry issue: [ACTION-PRODUCTION-EWT](https://parabol.sentry.io/issues/4299499003/?referrer=github_plugin)

**Comment by Dschoordsch** [comment]
Sentry issue: [ACTION-PRODUCTION-EWS](https://parabol.sentry.io/issues/4299499002/?referrer=github_plugin)

**Comment by Dschoordsch** [comment]
Reappeared due to a partial revert of the fix in #8472
